src/v2/schema/__snapshots__/application.test.js.snap

```diff
@@ -241,14 +241,14 @@ Object {
         "_uid": "local-cluster/app-02-uid",
         "name": "app02",
       },
-      Object {
-        "_uid": "local-cluster/29a848d6-3de8-11ea-9f0f-00000a100f99",
-        "name": "argo-app01",
-      },
       Object {
         "_uid": "remote-cluster/argo-app-02-uid",
         "name": "argo-app02",
       },
+      Object {
+        "_uid": "local-cluster/29a848d6-3de8-11ea-9f0f-00000a100f99",
+        "name": "argo-app01",
+      },
```

Comment:
Wondering why the order changed here.  Is descending order expected?

Reply:
Yes, it's because `argo-app01` has applicationSet `argo-set` so it will sort as if the name were `argo-set/argo-app01`.